src/modals/PasteYAML.tsx

```diff
@@ -27,7 +27,7 @@ type Props = {
   importYAML: (yamlString: string) => void;
 };
 
-export default ({ open, closeModal, importYAML }: Props) => {
+const PasteYAML = ({ open, closeModal, importYAML }: Props) => {
```

Comment:
I like the changes, should we (in the future) have named exports?
ex `export { PasteYAML}`
Wondering if you see benefits to this style